x-pack/auditbeat/module/system/_meta/fields.yml

```diff
@@ -4,7 +4,25 @@
     These are the fields generated by the system module.
   release: experimental
   fields:
-    - name: system.audit
-      type: group
+
+  - name: event
+    type: group
+    fields:
+    - name: origin
```

Comment:
@ruflin I think this would make a great addition to ECS for later. Maps well to the semantics of Filebeat 6.x `source`.

Reply:
Let's open issues in ECS and best with some value examples 👍